test/common/router/rds_impl_test.cc

```diff
@@ -510,6 +515,51 @@ name: foo_route_config
             route_config_provider_manager_->dumpRouteConfigs()->dynamic_route_configs().size());
 }
 
+TEST_F(RouteConfigProviderManagerImplTest, SameProviderOnTwoInitManager) {
+
```

Comment:
Nit: gratuitous space, aim for Google3 dense style.

Reply:
:+1: 
fixed